config/crd/bases/secrets.contentful.com_syncedsecrets.yaml

```diff
@@ -1,9 +1,10 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
@omegion how did you manage to keep it working with previous version?
can you compile for us?